ArchiSteamFarm/www/js/app.js

```diff
@@ -1041,6 +1041,7 @@ $(function () {
                 for (var prop in obj) {
                     if (obj.hasOwnProperty(prop)) {
                         var language = obj[prop];
+                        if (language === 'README.md') continue;
```

Comment:
I'd suggest to query against `*.json` instead.

Reply:
Or even better, change output translations to `strings.%locale%.json` and query against `strings.*.json`.

Reply:
Or I could change `strings.json` to `xx-XX,json`, then check for `aa-AA.json`. What do you think?

Reply:
Your call but I suggest `strings.*.json` instead since it clearly declares `strings.json` as being source file, while `en-US.json` is no longer clear whether it's automatically generated or not.

Reply:
It should be working with latest commit.